downloadDirectory/README.md

```diff
@@ -178,9 +147,15 @@ The file should have the same amount of lines as Human Pathway instances in `tes
 
 This step will match old <b>stableIdentifiers</b> to ones in the new format. The file contains two tab-seperated columns of stable IDs in the new (eg: `R-HSA-1234567`) and old (eg: `REACT_98765`) formats. The file should include stableIdentifiers for <b>all species</b> in Reactome, and should have approximately the same number of lines as <i>stableIdentifier</i> instances in `test_reactome`.
 
-<h4>gene_association.reactome</h4>
+<h4>GenerateGOAnnotationFile</h4>
+
+This step generates the 'gene_association.reactome' GO Annotation file. Information about the file format can be found <a href="https://www.ebi.ac.uk/GOA/newto">here</a>. This step will go through all <b>curated</b> ReactionlikeEvents that are in the database and generate GOA lines for a variety of instances pertaining to all 3 of the Gene Ontology annotation types: <b>Cellular Compartment</b>, <b>Molecular Function</b> and <b>Biological Process</b>.
+
+`Cellular Compartment`: It will evaluate all proteins (EntityWithAccessionedSequences) associated with a ReactionlikeEvent in the entire hierarchy.<br><br>
+`Molecular Function`: It will evaluate all proteins in the CatalystActivity attribute of the ReactionlikeEvent. More specifically it will annotate the proteins in the ActiveUnit or, if no ActiveUnit exists, the PhysicalEntity slots. The type of annotation produced for Molecular Function accessions depends on if any PubMed identifiers exist in the protein of interest. Those with a pmid will get a GOA entry that cites PubMed, while those that don't will receive the typical Reactome annotation.<br><br>
```

Comment:
There's a bunch of caveats as to whether proteins for a CatalystActivity get annotated when they are part of a complex or set.  Could these be discussed here or a link to another section with the details?

Reply:
Yea thats fair. I was trying to keep the README that is specific to DownloadDirectory a bit light. Maybe it would make more sense to have module-specific READMEs?